### ISSUE TITLE
No intellisense for some typings until tsconfig.json created

### ISSUE THREAD

**Issue by benjaminRomano** [Microsoft/vscode#835]
For some typings, intellisense does not start working until a tsconfig.json config file is included. Is this intended behavior?

Case 1:
Gulp typings without a tsconfig.json included. **Typings work**
![image](https://cloud.githubusercontent.com/assets/9221137/11484146/19c63ba8-9772-11e5-8617-2963199c5e87.png)

Case 2: 
Extjs typings without a tsconfig.json. **Typings don't work**
![image](https://cloud.githubusercontent.com/assets/9221137/11484179/405cc14c-9772-11e5-846b-f83c934935aa.png)

Case 3:
Extjs typings with a tsconfig.json.**Typings work**
![image](https://cloud.githubusercontent.com/assets/9221137/11484229/8734c920-9772-11e5-94f8-fad5592d0e8e.png)

**Comment by dbaeumer** [comment]
For TS files you either need a tsconfig.json file or use /// reference imports. See https://code.visualstudio.com/docs/languages/typescript

**Comment by EthraZa** [comment]
I tryed. 
I have typings/extjs/ExtJS.d.ts.
I have created a jsconfig.json at root level of my Sencha ExtJS 6 project folder:
~~~
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "ES5",
        "outDir": "out",
        "noLib": true,
        "sourceMap": true
    },
    "exclude": [
        "node_modules",
        "ext",
        "overrides",
        "resources",
        "sass",
        "build",
        "packages",
        "help"
    ]
}
~~~

When I open any js file I just get a popdown Warning saying:
~~~
Sorry, but there are too many JavaScript source files for VS Code. Consider using the exclude-property in jsconfig.json.
~~~
And no IntelliSense.

I have another web project that is not a Sencha MVC project, but uses ExtJS 4 on js files. In this project I get no warnings, but I get no IntelliSense as well.
